Sources/ImageFetcher/ImageFetcher.swift

```diff
@@ -39,25 +39,23 @@ extension NSImage {
 #else
 import UIKit
 #endif
-import DataOperation
 import DiskCache
 
-public final class ImageFetcher: ImageFetching {
+public actor ImageFetcher {
```

Comment:
Removal of `ImageFetching` conformance is temporary; just trying to get it to compile while focusing attention on this set of changes